__tests__/__snapshots__/users.spec.js.snap

```diff
@@ -29,6 +29,13 @@ Array [
 ]
```

Comment:
Why do you have changes in snapshots ?

Reply:
I have no idea why but I don't want to think it was husky